pkg/archive/parser.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2019 the Velero contributors.
+Copyright 2020 the Velero contributors.
```

Comment:
The copyright year can be removed entirely, that should help in keeping these up-to-date.

Reply:
Removing the year sounds much more maintainable. I'll be sure to change the lines to be

`Copyright the Velero contributors.`